docs/heasarc/heasarc.rst

```diff
@@ -11,292 +11,272 @@ This is a python interface for querying the
-         SEQ_ID      INSTRUMENT EXPOSURE ...    START_TIME        END_TIME                 SEARCH_OFFSET_            
-                                   s     ...       mjd              mjd                                              
-    ---------------- ---------- -------- ... ---------------- ---------------- --------------------------------------
-    RH701576N00          HRI       68154 ... 49704.3090856482 49724.6236226852  0.191 (187.2779228198,2.0524148595)\n
-    RP600242A01          PSPCB     24822 ... 48980.6468865741 48982.9284837963 34.237 (187.2779228198,2.0524148595)\n
-    RH700234N00          HRI       17230 ... 48629.2693055556 48632.4716782407  0.191 (187.2779228198,2.0524148595)\n
-
-Note that the :meth:`~astroquery.heasarc.HeasarcClass.query_region` converts
-the passed coordinates to the FK5 reference frame before submitting the query.
-
-
-Modifying returned table columns
---------------------------------
-
-Each table has a set of default columns that are returned when querying the
-database. You can return all available columns for a given mission by specifying
-the ``fields`` parameter in either of the above queries. For exampe:
+    >>> pos = SkyCoord.from_name('ngc 3783')
+    >>> tab = Heasarc.query_region(pos, table='numaster')
+    >>> tab['name', 'obsid', 'ra', 'dec'][:3].pprint()
+      name    obsid      ra      dec   
+                        deg      deg   
+    -------- -------- -------- --------
+    NGC_3783 80701617 174.7573 -37.7386
+    NGC_3783 60901023 174.7571 -37.7385
+    NGC_3783 60902005 174.7571 -37.7385
+
+To query a region around some position, specifying the search radius,
+we use `~astropy.units`:
 
 .. doctest-remote-data::
 
-    >>> table = heasarc.query_object(object_name='3c273', mission='rosmaster', fields='All')
-
-will return all available columns from the ``rosmaster`` mission table.
-Alternatively, a comma-separated list of column names can also be provided to
-specify which columns will be returned:
+    >>> from astroquery.heasarc import Heasac
+    >>> from astropy.coordinates import SkyCoord
+    >>> from astropy import units as u
+    >>> pos = SkyCoord('120 38', unit=u.deg)
+    >>> tab = Heasac.query_region(pos, table='chanmaster', radius=2*u.deg)
+    >>> tab['name', 'obsid', 'ra', 'dec'][:5].pprint()
+               name           obsid     ra      dec   
+                                       deg      deg   
+    ------------------------- ----- --------- --------
+                    ABELL 611  3194 120.23708 36.05722
+                   B2 0755+37   858 119.61750 37.78667
+    WISEA J080357.73+390823.1 28213 120.99060 39.13980
+        1RXS J075526.1+391111 13008 118.85875 39.18639
+     SDSS J080040.77+391700.5 18110 120.17000 39.28344
+
+If no radius value is given, a default that is appropriate
+for each table is used. You can see the value of the default
+radius values by calling `~~astroquery.heasarc.HeasarcClass._get_default_radius`,
```

Comment:
This is a private function, don't refer to it in the narrative documentation.

Reply:
I think this method can be useful to the user, and that is why it is documented. It was initially public, and  changed it to private after the first set of comments. I leaning towards making it public again and then referenced it here.